packages/terminal/tui/LICENSE.md

```
@@ -1308,11 +1296,14 @@ Repository: git+https://github.com/Morglod/tseep.git
 <!-- TYPE_DEPENDENCIES -->
 
 # Licenses of bundled types
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)